.vscode/settings.json

```diff
@@ -8,7 +8,7 @@
   // Controls if the editor should automatically close brackets after opening them
   "editor.autoClosingBrackets": false,
   // Controls whether the editor should render whitespace characters
-  "editor.renderWhitespace": true,
+  "editor.renderWhitespace": "all",
```

Comment:
Can you use the intermediate value that does not render whitespace between words?